modules/monitoring-enabling-alert-routing-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/enabling-alert-routing-for-user-defined-projects.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="enabling-alert-routing-for-user-defined-projects_{context}"]
+= Enabling alert routing for user-defined projects
+
+In {product-title} {product-version}, an administrator can enable alert routing for user-defined projects.
```

Comment:
@eromanova97 just a question
https://84593--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/enabling-alert-routing-for-user-defined-projects.html
`In OpenShift Container Platform Branch Build`, I think the Branch Build would be replaced by a actual version when the doc merged, but not sure, or the `{product-version}` is no need here, can you help to confirm?

Reply:
Yes, you are right :) it should populate the correct version once merged. But as you mention, it actually seems quite unnecessarry here, so I will remove the line. Thank you!